lib/connection.ts

```diff
@@ -323,25 +360,35 @@ export class Connection extends Entity {
 
   /**
    * Closes the amqp connection.
+   * @param options A set of options including a signal used to cancel the operation.
+   * When the abort signal is used, the local endpoint is indeed closed.
```

Comment:
`abort aignal` => `options.abortSignal`?

`indeed` not needed here. Can just be 'closed'.

Reply:
Fixed comments in https://github.com/amqp/rhea-promise/pull/66/commits/2487cae6bc7531d3c7f3675adb300980d97e315e